### DISCUSSION TITLE
Smart Meter missing in the integration

### DISCUSSION THREAD

**Discussion by g-zajac** [thomluther/anker-solix-api#120] [General]
Hi!

First of all thank you so much for such a great work, much appreciate! 
In short, I have added a system, details below, but Smart Meter is missing and looking how to add it in.

I have got Solarbank 2 Pro (v1.0.5.5) with expansion battery (v0.1.8.7) and Smart Meter (v0.0.0.29).
The SB is linked with the Smart Meter, power usage scenario set to “Self-Consumption”
All set in the Anker App (iOS ver 2.8.0) and working fine. I have set additional user account with different email and make it family member. With the additional email I have set HASS Anker integration (v1.3.2).

On the Anker integration I can see 2 devices: Solarbank 2 E1600 Pro and System Home but not Smart Meter.
Any suggestions, directions how to add the Smart Meter please?
Thanks!

**Comment by thomluther** [comment]
Hi @g-zajac 
You need to install the 2.0.0 beta as discussed [here ](https://github.com/thomluther/hacs-anker-solix/discussions/93)

**Reply by g-zajac** [reply]
just found it, thank you so much! I have installed the beta, got the smart meter in the system, working all good so far, will test.
